php_phongo_structs.h

```diff
@@ -204,25 +204,25 @@ typedef struct {
 	PHONGO_ZEND_OBJECT_PRE
 	mongoc_client_t    *client;
 	char               *command_name;
-	uint32_t            server_id;
+	int                 server_id;
```

Comment:
The server and cursor structs use `int`. I'm not sure where `uint32_t` came from.

I believe this is only a diff because I re-ordered the struct definitions to be alphabetical; however, it'd be a good idea to make them consistent.